core/tests/puppeteer-acceptance-tests/spec/practice-question-submitter-tests/suggest-medium-drag-and-drop-sort-question.spec.js

```diff
@@ -0,0 +1,192 @@
+    'is equal to ordering with one item at incorrect position');
+  const RESPONSE_ALL_OTHER_ANSWERS_ARE_WRONG = 'all other answers are wrong';
+  const SOLUTION_QUANTITY_ONLY_ONE = 'Only';
+
+  beforeAll(async function() {
+    const superAdmin = await userFactory.createNewSuperAdmin('superadm');
+    await superAdmin.assignRoleToUser('superadm', ROLE_CURRICULUM_ADMIN);
+
+    const skill = {
+      description: skillDescription,
+      reviewMaterial: 'how to perform skill',
+      difficulties: {
+        [DIFFICULTY_MEDIUM]: { rubricNotes: mediumDifficultyRubricNotes },
+        [DIFFICULTY_HARD]: { rubricNotes: hardDifficultyRubricNotes }
+      },
+      misconception,
+      questionCount: 3,
+    };
+    await superAdmin.createSkill(skill);
+    await superAdmin.createTopic({
+      name: topicName,
+      urlFragment: 'test-topic',
+      webTitleFragment: 'Between "topic name |" and "| Oppia"',
+      description: 'describe the topic',
+      thumbnail: testConstants.images.exampleTopicThumbnailImage,
+      metaContent: 'test',
+      assignedSkills: [skill],
+      subtopics: [{
+        title: 'Brief & understandable description for subtopic',
+        urlFragment: 'test-subtopic',
+        description: 'Detailed explanation of subtopic',
+        thumbnail: testConstants.images.exampleTopicThumbnailImage,
+        assignedSkills: [skill]
+      }],
+      diagnosticTestSkills: [skill],
+      isPublished: true
+    });
+
+    const topicId = await superAdmin.getTopicIdBy({ name: topicName });
+    await superAdmin.editClassroom({ topicId });
+
+    questionSubmitter = (
+      await userFactory.createNewQuestionSubmitter('questionsubmitter'));
+  }, DEFAULT_SPEC_TIMEOUT);
+
+  it('should suggest questions by selecting the difficulty to a lesson' +
+    ' in a topic', async function() {
+    await questionSubmitter.navigateToContributorDashboard();
+    await questionSubmitter.navigateToSubmitQuestions();
+    await questionSubmitter.expectSkillOpportunityToExist({
+      topicName, skillDescription, suggestable: true
+    });
+
+    await questionSubmitter.suggestQuestionForSkillOpportunity(
+      { topicName, skillDescription });
+    await questionSubmitter.expectQuestionDifficultyChoices(
+      {
+        difficulty: DIFFICULTY_MEDIUM,
+        rubricNotes: mediumDifficultyRubricNotes
+      },
+      {
+        difficulty: DIFFICULTY_HARD,
+        rubricNotes: hardDifficultyRubricNotes
+      });
+
+    await questionSubmitter.chooseDifficulty(DIFFICULTY_MEDIUM);
+    await questionSubmitter.expectChosenDifficultyToBe(
+      DIFFICULTY_MEDIUM);
+    await questionSubmitter.expectSkillRubricNotesToHave(
+      mediumDifficultyRubricNotes);
+    await questionSubmitter.expectQuestionTextSection();
+
+    await questionSubmitter.writeQuestionText(questionText);
+    await questionSubmitter.expectQuestionTextToBe(questionText);
+
+    await questionSubmitter.addInteraction();
```

Comment:
Let's push all of this (up to saveInteraction) into the questionSubmitter utility layer. The top-level should just be:

    `await questionSubmitter.addDragAndDropSortInteraction(['3', '1', '2'], true)`

where the last argument is for the "multiple items in same position" case. This makes the utility method reusable.

Reply:
The feature related to this review thread was removed.